### PR TITLE
feat: english code & link emails

### PR DESCRIPTION
**Pull request by danielgent**
:warning: *when ready to merge, run script `scripts/create-mjml-templates.sh` to upload these to dev* :warning: 

:exclamation: _this PR only does the code and link emails_ :exclamation: 
I've been manually pushing them and testing on https://github.com/sesamyab/login2/pull/455 to confirm it works the same

### CODE REVIEW

File: src/controllers/email.ts [hunk 1]
```diff
@@ -72,14 +74,27 @@ export async function sendCode(
   const templateString = await response.text();
 
   const language = client.tenant.language || "sv";
+  const locale = language === "sv" ? sv : en;
```
Comment (danielgent) [inline src/controllers/email.ts:77]: on dev at least most of our tenants don't have a language set so I'll fall back to swedish

File: src/controllers/email.ts [hunk 2]
```diff
@@ -72,14 +74,27 @@ export async function sendCode(
...
+  const sendCodeUniversalTemplate = engine.parse(templateString);
+  const sendCodeTemplateString = await engine.render(
+    sendCodeUniversalTemplate,
+    {
+      ...locale,
+      code,
+      vendorName: client.name,
+      logo,
+      supportUrl: client.tenant.support_url || "https://support.sesamy.com",
+      buttonColor: client.tenant.primary_color || "#7d68f4",
```
Comment (danielgent) [inline src/controllers/email.ts:94]: lol. I didn't think about this when writing the ticket

On the first round of `liquidjs` *every* `{{}}` will be replaced

So (see below!) I feed the same vars in twice.

There'll be a cleaner way I'm sure but this works :muscle:

**Review by markusahlstrand — APPROVED** [review]
Look nice! Just as I imagined it :)

**Comment by danielgent** [comment]
> Look nice! Just as I imagined it :)

It's like AI but you don't have to type  :upside_down_face:  (for now!)

**Pull request merged**